framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/CopiableText/CopiableText.module.scss

```diff
@@ -1,10 +1,11 @@
 .clickable {
 	cursor: pointer;
+	margin-left: 8px;
 }
 
 .clickableRow {
 	padding: 0.5px;
-	height: auto;
+	max-width: 500px;
```

Comment:
We can't assign max-width inside the component. It must inherit the width of the container. If container is 600px and content is also long it should just stretch to available width.

Reply:
the problem is that if we keep width relative to parent container, it always wraps the text in a way that on hover it flickers

Reply:
I have given max-width as 95% of the container and 5% to icon so that they have their own space within the container and also the width can vary with the width of parent container